File: PartnerApi/Swift/b2_authorize_account.swift
import Foundation

let masterApplicationKeyId = ""  // Obtained from your B2 account page.
let masterApplicationKey = ""  // Obtained from your B2 account page.

let base64EncodedString = Data("\(masterApplicationKeyId):\(masterApplicationKey)".utf8)
  .base64EncodedString()

var request = URLRequest(
  url: URL(string: "https://api.backblazeb2.com/b2api/v4/b2_authorize_account")!)
request.setValue("Basic \(base64EncodedString)", forHTTPHeaderField: "Authorization")
let (data, response) = try await URLSession.shared.data(for: request)

print(String(decoding: data, as: UTF8.self))

File: PartnerApi/PHP/b2_authorize_account.php
<?php

$master_application_key_id = ''; // From your B2 account page
$master_application_key = ''; // From your B2 account page

$ch = curl_init('https://api.backblazeb2.com/b2api/v4/b2_authorize_account');
curl_setopt_array($ch, [
    CURLOPT_RETURNTRANSFER => true,
    CURLOPT_HTTPHEADER     => [
        'Authorization: Basic ' . base64_encode("$master_application_key_id:$master_application_key"),
    ],
]);

echo curl_exec($ch), PHP_EOL;
